src/Symfony/Component/Messenger/EventListener/StopWorkerOnSignalsListener.php

```diff
@@ -18,6 +18,8 @@
 /**
  * @author Tobias Schultze <http://tobion.de>
  * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ *
+ * @deprecated since Symfony 6.3, use the {@see \Symfony\Component\Console\Command\SignalableCommandInterface} instead
```

Comment:
deprecations are only for the feature branch (6.4 today)
should we split the PR in two parts, one for 6.3 the other for 6.4?

Reply:
Makes sense to me, see #50997